frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -248,6 +248,23 @@
     margin-left: $pad-small;
   }
 
+  &__macsettings-dropdown {
```

Comment:
wonder if we can combine this with policy pass fail dropdown and clean up the css/use a svg icon

we can save css cleanup for a later revision